src/main/java/org/sagebionetworks/web/client/view/SearchViewImpl.java

```diff
@@ -377,22 +375,19 @@ private Panel getResult(int i, Hit hit) {
 		createdByBadge.configure(getSearchUserId(hit.getCreated_by()));
 		UserBadge modifiedByBadge = ginInjector.getUserBadgeWidget();
 		modifiedByBadge.configure(getSearchUserId(hit.getModified_by()));
-
+		attributionPanel.addStyleName("hitattribution");

```

Comment:
Each part ("Created by X Modified By Y") used to have this css class name.  Now apply to entire attribution panel.  And change the style so that it doesn't make the text green, but rather italicized and with a proper margin to space out the hits.